Package.swift

```diff
@@ -4,7 +4,7 @@ import PackageDescription
 let package = Package(
     name: "Crypto",
     products: [
-        .library(name: "Crypto", targets: ["Crypto"]),
+        .library(name: "Crypto", targets: ["Crypto", "libbcrypt"]),
```

Comment:
I think this change can be removed, we don't want to publicize `libbcrypt`. That's an implementation detail.